client/src/js/samples/components/LabelEditor/CreateLabel.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { connect } from "react-redux";
+import { InputGroup, InputLabel, Input, Button, InputError, Modal, ModalHeader, ModalBody } from "../../../base";
+import { ColorSelector } from "./ColorSelector";
+
+const getInitialState = ({ labelName, color, description, errorName, errorColor }) => ({
```

Comment:
Do not need to derive state from props. Form will be empty initially.

`constructor` method is not needed for class.